pkg/core/interop_system.go

```diff
@@ -272,7 +274,11 @@ func runtimeNotify(ic *interop.Context, v *vm.VM) error {
 
 // runtimeLog logs the message passed.
 func runtimeLog(ic *interop.Context, v *vm.VM) error {
-	msg := fmt.Sprintf("%q", v.Estack().Pop().Bytes())
+	state := v.Estack().Pop().Bytes()
+	if len(state) > MaxNotificationSize {
+		return fmt.Errorf("message length shouldn't exceed %v", MaxNotificationSize)
+	}
+	msg := fmt.Sprintf("%q", state)
```

Comment:
C# node now enforces this slice to be strict UTF-8 and throws an exception otherwise: https://github.com/neo-project/neo/pull/1710
https://dotnetfiddle.net/mL1U6d
`utf8.IsValid` check should be enough.

Reply:
Added this check. Also fixed the same issue in `System.Runtime.Notify`.